systemtests/go.mod

```diff
@@ -0,0 +1,162 @@
+module cosmossdk.io/systemtests
```

Comment:
Could you make a PR here: https://github.com/cosmos/vanity
cc @alpe

Reply:
https://github.com/cosmos/vanity/pull/68

Reply:
sorry, missed this. Thanks for the PR